src/seahawk_rov/seahawk_rov/thrust_box_servo.py

```diff
@@ -79,4 +55,4 @@ def __init__(self, node, i2c):
 
     def receive_thruster(self, message:Float32MultiArray):
         for thruster in self.thruster_map:
-            self.kit.servo[thruster].angle = int(lerp(-1.0, 1.0, 0, 3000, clamp(message.data[thruster], -1.0, 1.0)))
+            self.kit.servo[thruster].angle = int(seahawk_rov.lerp(-1.0, 1.0, 0, 3000, seahawk_rov.clamp(message.data[thruster], -1.0, 1.0)))
```

Comment:
Perhaps it would be a good idea to change `0` & `3000` to `PWM_MIN` and `PWM_MAX` constants

Reply:
i un-genericized it for you